### PR TITLE
refactor(chart): improved `chart` range selector and style

### LINKED ISSUE #427: Refactor `Chart` component example
### Subject

`Chart`

### Description

There are few things that could be improved in the `Line Chart` component example.

### Are you willing to participate in the development of this component and create a pull request?

Yes

### PR DESCRIPTION
**Pull request by taroj1205**
Closes #427

## Description

I have added a better range picker for the date and refactored the style.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: contents/data-display/charts/weekly-temperature-chart/metadata.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   },
   "options": {
     "container": {
-      "p": "md"
+      "minH": "3xl"
     }
```
Comment (illionillion) [inline contents/data-display/charts/weekly-temperature-chart/metadata.json:13]: How about making utils.ts appear before locations.ts?

```suggestion
    },
    "files": {
      "order": ["index.tsx", "utils.ts", "location.ts"]
    }
```

File: contents/data-display/charts/weekly-temperature-chart/utils.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  const data = (await response.json()) as Array<{
+    latitude: number
+    longitude: number
+    hourly: {
+      time: string[]
+      temperature_2m: number[]
+    }
+  }>
+
+  const locationKeys = ["Tokyo", "Osaka", "Hokkaido", "Okinawa"]
```
Comment (illionillion) [inline contents/data-display/charts/weekly-temperature-chart/utils.ts:28]: What about importing `locationKeys` from `locations.ts`?

File: contents/data-display/charts/weekly-temperature-chart/metadata.json (hunk 1 above)
Comment (taroj1205) [reply contents/data-display/charts/weekly-temperature-chart/metadata.json:16]: Done.

File: contents/data-display/charts/weekly-temperature-chart/utils.ts (hunk 2 above)
Comment (taroj1205) [reply contents/data-display/charts/weekly-temperature-chart/utils.ts]: Done.

File: contents/data-display/charts/weekly-temperature-chart/index.tsx [hunk 3]
```diff
@@ -25,48 +30,60 @@ const WeeklyTemperatureChart: FC = () => {
...
-    )
-
-    return formattedData
+    try {
+      if (!dateRange[0] || !dateRange[1]) return []
+
+      const startDateString = formatDate(dateRange[0])
+      const endDateString = formatDate(dateRange[1])
+
+      return getWeeklyTemperatureData(startDateString, endDateString)
```
Comment (illionillion) [inline contents/data-display/charts/weekly-temperature-chart/index.tsx:45]: Don't you use await?

```suggestion
      return await getWeeklyTemperatureData(startDateString, endDateString)
```

Comment (taroj1205) [reply contents/data-display/charts/weekly-temperature-chart/index.tsx]: Fixed.

**Review by illionillion — APPROVED** [review]
LGTM

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**